source/_integrations/light.template.markdown

```diff
@@ -145,6 +185,8 @@ For example, you would replace
 with this equivalent that returns `true`/`false` and never gives an unknown
 result:
 {% raw %}`{{ is_state('switch.source', 'on') }}`{% endraw %}
+Passing `transition` and `brightness` parameters along with `*_color`, `color_temp`, `white_value` or `effect` is supported.
```

Comment:
Explain that "transition" will be passed as named parameter `transition` to the scripts.
Explain that "brightness" will be passed as named parameter `brightness` to the scripts.

Please also update the documentation for `set_level` to explain that the script will only be called if the `turn_on` call only has brightness, and optionally, transition.

Reply:
done